### ISSUE TITLE
Search not proper working in a language different from English.

### ISSUE THREAD

**Issue by oscarAguayo** [squidfunk/mkdocs-material#1675]
__I checked that...__

- [x] ... the documentation does not mention anything about my problem
- [x] ... the problem doesn't occur with the default MkDocs template
- [x] ... the problem is not in any of my customizations (CSS, JS, template)
- [x] ... there are no open or closed issues that are related to my problem

## Description

If you change the`theme .language: != en`, the search not working as expected. I tested it in `fr` and `es`, even I tried with `plugins .search .lang` and not working. I think the same will happen with other languages.

### Expected behavior

Write a text, for example: "Welcome" (its a default title in a new project) in the search box, and the mkdocs find it.

### Actual behavior

If you search the text "welcome" (without quotes) into the search box, in a new project, the search shows: "No matching documents." or in Spanish "No se encontraron documentos". Even when the first document has a "Welcome to MkDocs" in the title.

![image](https://user-images.githubusercontent.com/3291694/81193459-07a49080-8f81-11ea-9475-24254860dcf1.png)

### Steps to reproduce the bug

1. Create a new project ´mkdocs new example`
2. Set in `mkdocs.yml`
```
...
# Configuration
theme:
  name: material
  language: es # or fr
  palette:
    primary: indigo
    accent: indigo
  font:
    text: Roboto
    code: Roboto Mono
...
```
3. Go into new folder `cd example` and run local server with `mkdocs serve`.
4. Open in browser the url ´http://127.0.0.1:8000/´
5. In the search box type "welcome" (without quotes)
6. The search shows: "No matching documents." or in spanish "No se encontraron documentos", even when the first document has a "Welcome to MkDocs" in the title.

### Package versions

* Python: Python 3.7.3
* MkDocs: mkdocs, version 1.1 from .../mkdocs/env/lib/python3.7/site-packages/mkdocs (Python 3.7)
* Material: Version: 5.1.5

### Project configuration

``` yaml
site_name: My Docs

# Configuration
theme:
  name: material
  language: es # or fr
  palette:
    primary: indigo
    accent: indigo
  font:
    text: Roboto
    code: Roboto Mono
```

### System information

* OS: macOS Mojave, Version 10.14.5
* Browser: Google Chrome, Versión 81.0.4044.129 (Build oficial) (64 bits)

**Comment by squidfunk** [comment]
Duplicate of #1603 – also related to stemming:

![Ohne Titel](https://user-images.githubusercontent.com/932156/81471718-55283400-91f3-11ea-8023-2ab207482af2.gif)

Currently not fixable without losing type ahead (see my comments in the linked issue). I'm thinking of dropping stemming entirely as it seems to cause more problems than upsides.

**Comment by squidfunk** [comment]
Closing as duplicate.

**Comment by squidfunk** [comment]
As noted in https://github.com/squidfunk/mkdocs-material/issues/1745#issuecomment-640206622, search should now yield better results/behavior for languages other than English as of 36e1e3d. The change is released as part of 5.2.3.
